src/components/accessories/patientExams/examRowTable/ExamRowTable.tsx

```diff
@@ -56,7 +61,7 @@ const ExamRowTable: FC<IEditableTableProps> = ({
             size="small"
             aria-label="results table"
           >
-            <TableHead>
+            {/* <TableHead>
```

Comment:
why to comment out instead of just removing the piece of code?

Reply:
Because I don't agree with you to remove the header. So I just hide it.

Reply:
You just said "you think" we can omit the header, and it not explains why we could omit it. I was working on other things, so in order not to lose the thread, I just applied your suggestion. But in the future I'll ask you why you want me to apply changes.

Reply:
I removed it

Reply:
ok, thanks, it LGTM. 
The why I said it can be omitted it's because it doesn't add any meaning to the results list (that is self-explained) and makes the UI unnecessarily verbose.